clang/test/Driver/cxx-static-destructors.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -### -c -fc++-static-destructors=all %s 2>&1 | FileCheck --check-prefix ALL %s
+// RUN: %clang -### -c -fc++-static-destructors %s 2>&1 | FileCheck --check-prefix ALL %s
+// RUN: %clang -### -c -fc++-static-destructors=none %s 2>&1 | FileCheck --check-prefix NONE %s
```

Comment:
You can add -fc++-static-destructors before -fc++-static-destructors=none to enhance the test to show we use the `getLastArg` semantics